lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+'''
+Unit Tests for the manage.py command rename_user
+'''
+from django.core.management.base import CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from .. import rename_user
+import sys
+from StringIO import StringIO
+
+
+class TestUsernameRenameCommand(unittest.TestCase):
+    '''
+    Class which contains various unit tests for the manage.py rename_user command line tool
+    '''
+    @classmethod
+    def setUpClass(cls):
+        client = MongoClient()
+        # chooses to temporarily write to the forum database
+        cls.db = client.forum
+        # maintain arrays of mongo data so that we can delete them when the tests are done
+        cls.mongo_user_ids = []
+        cls.mongo_comment_ids = []
+        cls.sql_user = UserFactory.create()
+        cls.rename_cmd = rename_user.Command()
+        cls.seed_mongo_comment()
+        cls.seed_mongo_user()
+
+    @classmethod
+    def seed_mongo_comment(cls):
+        '''
```

Comment:
Use double-quotes for docstrings, per the style guide [1].

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines